cliquet/scripts/cliquet.py

```diff
@@ -2,10 +2,17 @@
 import argparse
 import sys
 import textwrap
+import warnings
 
 from pyramid.paster import bootstrap
 
 
+def deprecated_init(env):
+    message = '"cliquet init" is deprecated. Use "cliquet migrate" instead.'
```

Comment:
migrate sounds weird for an initializer. I would have expected to have two commands: init and migrate (not related)